skema/program_analysis/CAST2FN/ann_cast/to_gromet_pass.py

```diff
@@ -117,12 +117,12 @@ def retrieve_name_id_pair(node):
     if isinstance(node, AnnCastName):
         return (node.name, node.id)
     if isinstance(node, AnnCastAttribute):
-        if isinstance(node.value, (AnnCastAttribute, AnnCastName)):
+        if isinstance(node.value, (AnnCastAttribute, AnnCastName, AnnCastCall)):
             return retrieve_name_id_pair(node.value)
         return (node.value, node.value.id)
     if isinstance(node, AnnCastCall):
         return retrieve_name_id_pair(node.func)
-
+    return ("",-1)
```

Comment:
Will returning this make things harder to debug in the future?  Should we instead raise and exception and bring things to a screeching halt?